### ISSUE TITLE
Esc key on hardware keyboards

### ISSUE THREAD

**Issue by CyberShadow** [connectbot/connectbot#433]
Hitting the Esc key on a physical keyboard should send the ANSI escape code, instead of working like the Android "back" button.

I fixed this in ConnectBot VX, here: https://github.com/vx/connectbot/commit/d37065223693cf0631cf8522b6b0bf87518a5c0d

**Comment by dredmorbius** [comment]
What's the status on this?  It's pretty much the killer ConnectBot feature.  Or rather, killing feature, as the esc / back behaviour kills ConnectBot entirely for me.

**Comment by brianjmurrell** [comment]
I agree with @dredmorbius.  This issue is a showstopper for me.  I am so tired of having to keep reopening the app because I press the ESC key on my hardware keyboard.
